src/metabase/api/embed.clj

```diff
@@ -314,7 +315,8 @@
   [{{:keys [token export-format]} :params, :keys [query-params]} respond raise]
   {export-format dataset-api/ExportFormat}
   (dataset-api/as-format-async export-format respond raise
-    (run-query-for-unsigned-token-async (eu/unsign token) (m/map-keys keyword query-params), :constraints nil)))
+    (fn [f]
+      (run-query-for-unsigned-token-async (eu/unsign token) (m/map-keys keyword query-params), :constraints nil, :data-fn f))))
```

Comment:
This line is getting a little long. Would you mind splitting it up into multiple lines, please?